alf/trainers/evaluator.py

```diff
@@ -74,10 +74,15 @@ def __init__(self, config: TrainerConfig, conf_file: str):
                       pre_configs, num_envs, config.root_dir, seed))
             self._worker.start()
         else:
-            self._env = create_environment(
-                for_evaluation=True,
-                num_parallel_environments=num_envs,
-                seed=seed)
+            if config.shared_train_eval_env:
```

Comment:
Need to set the step_type in the replay buffer just before evaluation started to StepType.LAST.

Reply:
Also need to set the next step type to FIRST